lib/jsdom/living/interfaces.js

```diff
@@ -19,7 +19,7 @@ const generatedInterfaces = [
   require("./generated/Element"),
   require("./generated/DocumentFragment"),
   require("./generated/Document"),
-  require("./generated/Document"),
```

Comment:
I’ve&nbsp;extracted this&nbsp;fix into&nbsp;<https://github.com/jsdom/jsdom/pull/2753>.